client/src/app/services/wordList.service.ts

```diff
@@ -12,6 +12,8 @@ export class WordListService {
   private dataStore: {
     wordList: Word[]
   };
+  private dataStashToUpDate: Word[] = [];
+  private dataStashToRemove: string[] = [];
```

Comment:
Myślę że nie ma w tym momencie potrzeby robienia tego, bo jest taki middleware do reduxa, który się tym zajmuje, i przy każdej zmianie stanu zapisuje stan w local storage. Więc to zaimplementujemy dość szybko jak już będzie redux działał